Core/Tests/Sources/Model/EnhancedLocationUpdateTests.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import CoreLocation
+import XCTest
+@testable import Core
+
+class EnhancedLocationUpdateTests: XCTestCase {
+
+    func testEnhancedLocationUpdateMessageDecodesNullBatteryLevel() throws {
```

Comment:
After merging that Android PR https://github.com/ably/ably-asset-tracking-android/pull/343 the battery level will be missing from the message JSON so maybe let's create a test for the scenario in which there is no battery level in the JSON? :thinking:

Reply:
Same reply as https://github.com/ably/ably-asset-tracking-cocoa/pull/146#discussion_r638056120